### PR TITLE
backend: update highest and lowest symbol prices in real time

### PR DESCRIPTION
**Pull request by thminhVN**
#### What's this PR do?

- [x] Technical solution for update highest and lowest symbol prices in real time

### CODE REVIEW

File: Engineering/Backend/Update highest and lowest symbol prices in real time.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+tags: engineering/backend, nghenhan, hedge-foundation
```
Comment (zlatanpham) [inline Engineering/Backend/Update highest and lowest symbol prices in real time.md:2]:
```suggestion
tags: engineering/backend, nghenhan, hedge-foundation, binance, web-socket
```

File: Engineering/Backend/Update highest and lowest symbol prices in real time.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+author: Minh Tran
+github_id: thminhVN
+date: 2023-05-07
+---
+
+# Introduction
```
Comment (zlatanpham) [inline Engineering/Backend/Update highest and lowest symbol prices in real time.md:8]: As the file name will turn to be H1, can you down level all headings by 1?

**Review by zlatanpham — APPROVED** [review]
Few nits, otherwise, LGTM

File: Engineering/Backend/Update highest and lowest symbol prices in real time.md (hunk 1 above)
Comment (thminhVN) [reply Engineering/Backend/Update highest and lowest symbol prices in real time.md]: Added

**Review by monotykamary — APPROVED** [review]
LGTM

**Pull request merged**